### PR TITLE
GG-31130 Detaregion metrics without description

### PR DESCRIPTION
**Pull request by vmalin**

### CODE REVIEW

File: modules/core/src/main/java/org/apache/ignite/mxbean/DataRegionMetricsMXBean.java [hunk 1]
```diff
@@ -124,6 +124,26 @@
     @MXBeanDescription("Offheap used size in bytes.")
     @Override public long getOffheapUsedSize();
 
+    /** {@inheritDoc} */
+    @MXBeanDescription("Gets memory page size.")
```
Comment (vmalin) [inline modules/core/src/main/java/org/apache/ignite/mxbean/DataRegionMetricsMXBean.java:128]: _The size of the memory page in bytes._

File: modules/core/src/main/java/org/apache/ignite/mxbean/DataRegionMetricsMXBean.java [hunk 2]
```diff
@@ -124,6 +124,26 @@
...
+    @Override public int getPageSize();
+
+    /** {@inheritDoc} */
+    @MXBeanDescription("Gets total size of pages loaded to the RAM. When persistence is disabled, this metric is equal TotalAllocatedSize.")
```
Comment (vmalin) [inline modules/core/src/main/java/org/apache/ignite/mxbean/DataRegionMetricsMXBean.java:132]: _The_ total size of pages loaded to the RAM. When persistence is disabled, this metric is equal _to_ TotalAllocatedSize.

File: modules/core/src/main/java/org/apache/ignite/mxbean/DataRegionMetricsMXBean.java [hunk 3]
```diff
@@ -124,6 +124,26 @@
...
+    @Override public long getPhysicalMemorySize();
+
+    /** {@inheritDoc} */
+    @MXBeanDescription("Gets used checkpoint buffer size in pages.")
```
Comment (vmalin) [inline modules/core/src/main/java/org/apache/ignite/mxbean/DataRegionMetricsMXBean.java:136]: _The_ used checkpoint buffer size in pages.

File: modules/core/src/main/java/org/apache/ignite/mxbean/DataRegionMetricsMXBean.java [hunk 4]
```diff
@@ -124,6 +124,26 @@
...
+    @Override public long getUsedCheckpointBufferPages();
+
+    /** {@inheritDoc} */
+    @MXBeanDescription("Gets used checkpoint buffer size in bytes.")
```
Comment (vmalin) [inline modules/core/src/main/java/org/apache/ignite/mxbean/DataRegionMetricsMXBean.java:140]: _The_ used checkpoint buffer size in pages.

File: modules/core/src/main/java/org/apache/ignite/mxbean/DataRegionMetricsMXBean.java [hunk 5]
```diff
@@ -124,6 +124,26 @@
...
+    @Override public long getUsedCheckpointBufferSize();
+
+    /** {@inheritDoc} */
+    @MXBeanDescription("Gets checkpoint buffer size in bytes.")
```
Comment (vmalin) [inline modules/core/src/main/java/org/apache/ignite/mxbean/DataRegionMetricsMXBean.java:144]: _The_ checkpoint buffer size in bytes.

Comment (a-polyakov) [reply modules/core/src/main/java/org/apache/ignite/mxbean/DataRegionMetricsMXBean.java]: The checkpoint buffer size in **bytes.**

File: modules/core/src/main/java/org/apache/ignite/mxbean/DataRegionMetricsMXBean.java (hunk 4 above)
Comment (a-polyakov) [reply modules/core/src/main/java/org/apache/ignite/mxbean/DataRegionMetricsMXBean.java]: The used checkpoint buffer size in **bytes**.

**Pull request merged**